src/controllers/queryRunner.ts

```diff
@@ -160,6 +162,18 @@ export default class QueryRunner {
                     querySelection: selection
                 };
 
+                const doc = await this._vscodeWrapper.openTextDocument(this._vscodeWrapper.parseUri(this._ownerUri));
+                let queryString = doc.getText();
```

Comment:
personally I would put the `getText` call in the `else` of the below conditions.  It looks like if there is a selection we'll toss out the return value, which causes an unnecessary, potentially expensive, IPC to the main process.